src/base/packer.cpp

```diff
@@ -42,8 +42,40 @@ namespace cubpacking
   //
   // static function declarations
   //
+  static void fatal_error_log (const char *ptr, const char *endptr, const size_t amount);
+
   static void check_range (const char *ptr, const char *endptr, const size_t amount);
 
+  static void fatal_error_log (const char *fmt, ...)
+  {
+    const char *msglog;
+    char msg[LINE_MAX];
+
+    va_list ap;
+
+    va_start (ap, fmt);
+
+    /* call er_set() to print call stack to the log */
+    vsnprintf (msg, LINE_MAX, fmt, ap);
+    er_set (ER_FATAL_ERROR_SEVERITY, ARG_FILE_LINE, ER_NET_DATASIZE_MISMATCH, 1, fmt);
```

Comment:
ER_NET_DATASIZE_MISMATCH requires two integer sizes.

Reply:
I changed error code in the next commit.